docs/internals/mailing-lists.txt

```diff
@@ -116,3 +65,42 @@ by developers and interested community members.
 .. _django-updates mailing archive: https://groups.google.com/g/django-updates
 .. _django-updates subscription email address: mailto:django-updates+subscribe@googlegroups.com
 .. _django-updates posting email: mailto:django-updates@googlegroups.com
+
+Archived mailing lists
+======================
+
+The following mailing lists are archived and no longer active. These are still
+available as a historic resource.
```

Comment:
To avoid suggesting importance.
More: https://www.merriam-webster.com/grammar/everything-youve-ever-wanted-to-know-about-historic-and-historical
```suggestion
available as a historical resource.
```